Mathlib/Algebra/Group/TypeTags.lean

```diff
@@ -132,6 +132,10 @@ instance [h: Infinite α] : Infinite (Additive α) := h
 
 instance [h: Infinite α] : Infinite (Multiplicative α) := h
 
+instance [h : DecidableEq α] : DecidableEq (Multiplicative α) := h
+
+instance [h : DecidableEq α] : DecidableEq (Additive α) := h
```

Comment:
I was going to suggest that these should be implemented via `decidable_of_iff` without defeq abuse, but I guess it's plausible that this is more convenient when exploiting defeqs elsewhere.

bors d+

Please merge once CI is happy.